src/sentry/runner/commands/backup.py

```diff
@@ -2,54 +2,198 @@
+    default="",
+    type=str,
+    help="An optional array of organizations to include. "
+    "If this option is not set, all encountered organizations are imported. "
+    "Users not members of at least one organization in this set will not be imported.",
+)
+@click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
+@configuration
+def import_organizations(src, filter_org_slugs, merge_users, silent):
+    """
+    Import the Sentry organizations, and all constituent Sentry users, from an exported JSON file.
+    """
+
+    import_in_organization_scope(
+        src,
+        flags=ImportFlags(merge_users=merge_users),
+        org_filter=parse_filter_arg(filter_org_slugs),
+        printer=(lambda *args, **kwargs: None) if silent else click.echo,
+    )
+
+
+@import_.command(name="global")
+@click.argument("src", type=click.File("rb"))
+@click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
+@configuration
+def import_global(src, silent):
+    """
+    Import all Sentry data from an exported JSON file.
+    """
+
+    import_in_global_scope(
+        src,
+        printer=(lambda *args, **kwargs: None) if silent else click.echo,
+    )
+
+
+@click.group(name="export")
+def export():
+    """Exports core data for the Sentry installation."""
+
+
+@export.command(name="users")
 @click.argument("dest", default="-", type=click.File("w"))
 @click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
 @click.option(
-    "--indent", default=2, help="Number of spaces to indent for the JSON output. (default: 2)"
+    "--indent",
+    default=2,
+    type=int,
+    help="Number of spaces to indent for the JSON output. (default: 2)",
+)
+@click.option(
+    "--filter_usernames",
+    default="",
+    type=str,
+    help="An optional array of users to include. "
+    "If this option is not set, all encountered users are imported.",
 )
-@click.option("--exclude", default=None, help="Models to exclude from export.", metavar="MODELS")
 @configuration
-def export(dest, silent, indent, exclude):
-    """Exports core data for the Sentry installation."""
+def export_users(dest, silent, indent, filter_usernames):
+    """
+    Export all Sentry users in the JSON format.
+    """
+
+    export_in_user_scope(
+        dest,
+        indent=indent,
+        user_filter=parse_filter_arg(filter_usernames),
+        printer=(lambda *args, **kwargs: None) if silent else click.echo,
+    )
+
+
+@export.command(name="organizations")
+@click.argument("dest", default="-", type=click.File("w"))
+@click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
+@click.option(
+    "--indent",
+    default=2,
+    type=int,
+    help="Number of spaces to indent for the JSON output. (default: 2)",
+)
+@click.option(
+    "--filter_org_slugs",
+    default="",
+    type=str,
+    help="An optional array of organizations to include. "
```

Comment:
might be good to add that this is in the form of a comma separated string

Reply:
Done.

Reply:
maybe not pushed yet?